### PR TITLE
[DX] Store passwords in plaintext in test environment

### PR DESCRIPTION
**Pull request by coldic3**
| Q               | A                                                            |
|-----------------|--------------------------------------------------------------|
| Branch?         | 1.13 |
| Bug fix?        | no                                                       |
| New feature?    | no                                                       |
| BC breaks?      | no                                                       |
| Deprecations?   | no |
| Related tickets | |
| License         | MIT                                                          |

The goal of this PR is to speed up our tests by using plaintext as a password. Thanks to this, there is no hashing algorithm that slows down every single test where a user has the be logged in.

### CODE REVIEW

**Review by Rafikooo — APPROVED** [review]

File: src/Sylius/Bundle/UserBundle/Resources/config/app/config.yml [hunk 1]
```diff
@@ -4,6 +4,7 @@
 security:
     password_hashers:
         argon2i: argon2i
+        plaintext: plaintext
```
Comment (NoResponseMate) [inline src/Sylius/Bundle/UserBundle/Resources/config/app/config.yml]: I'd be for removing this line from here and adding it only in test contexts to not have this hasher present by default.

File: composer.json [hunk 2]
```diff
@@ -73,7 +73,7 @@
         "sylius-labs/doctrine-migrations-extra-bundle": "^0.1.4 || ^0.2",
         "sylius-labs/polyfill-symfony-event-dispatcher": "^1.0.1",
         "sylius-labs/polyfill-symfony-framework-bundle": "^1.0 || ^1.1",
-        "sylius-labs/polyfill-symfony-security": "^1.1",
+        "sylius-labs/polyfill-symfony-security": "dev-master",
```
Comment (GSadee) [inline composer.json]: You can revert this change as the new version of polyfill has been released: https://github.com/SyliusLabs/PolyfillSymfonySecurity/releases/tag/v1.1.2

File: src/Sylius/Component/User/composer.json [hunk 3]
```diff
@@ -35,7 +35,7 @@
     "require": {
         "php": "^8.0",
         "doctrine/collections": "^1.6",
-        "sylius-labs/polyfill-symfony-security": "^1.1",
+        "sylius-labs/polyfill-symfony-security": "dev-master",
```
Comment (GSadee) [inline src/Sylius/Component/User/composer.json]: As above, revert this change

**Review by jakubtobiasz — APPROVED** [review]

**Review by NoResponseMate — APPROVED** [review]

**Comment by jakubtobiasz** [comment]
@coldic3 the PR is marked as WIP, is the status up-to-date?

**Review by jakubtobiasz — APPROVED** [review]

**Comment by coldic3** [comment]
> @coldic3 the PR is marked as WIP, is the status up-to-date?

@jakubtobiasz The PR is ready :+1: Forgot to remove WIP status.

**Comment by jakubtobiasz** [comment]
Thank you, Kevin! :1st_place_medal:
